packages/api/src/schemas/envConfig.json

```diff
@@ -6,6 +6,39 @@
     "$schema": {
       "type": "string"
     },
+    "auth": {
+      "type": "object",
+      "description": "Existing tab AAD app configuration.",
+      "properties": {
+        "clientId": {
+          "type": "string",
+          "description": "The client id of existing AAD app for tab.",
+          "minLength": 1
+        },
+        "clientSecret": {
+          "type": "string",
+          "description": "The client secret of existing AAD app for tab.",
+          "minLength": 1
+        },
+        "objectId": {
+          "type": "string",
+          "description": "The object id of existing AAD app for tab.",
+          "minLength": 1
+        },
+        "oauth2PermissionScopeId": {
```

Comment:
Discussed with @adashen , we would better to call it as "accessAsUserScopeId"

Reply:
updated, thanks for the suggestion.